test/TestCases/VctrBaseMemberFunctions.cpp

```diff
@@ -189,7 +189,7 @@ TEST_CASE ("forEach", "[VctrBaseMemberFunctions]")
     auto strings = loremIpsum;
 
     // In place for each function
-    strings.forEach ([] (std::string& s) { for (auto& c : s) c = std::toupper (c); });
+    strings.forEach ([] (std::string& s) { for (auto& c : s) c = char (std::toupper (c)); });
```

Comment:
aaah to upper.. I just googled wondering what a toupper is 😄 strange the standard didn't name this `std::to_upper()`..

Reply:
I think this is a super ancient C function (which also explains the int return type) that just made its way to the C++ `std` namespace. Nothing that I'd consider for usage in production code but here it was suitable to create a compact test case

Reply:
Makes sense!